_pages/about-us/offices/san-francisco.md

```diff
@@ -15,8 +15,7 @@ title: San Francisco
       <tr>
         <td class="col-key"><strong>Points of contact</strong></td>
         <td class="col-value">
-          Your best bet is the <a href="https://gsa-tts.slack.com/archives/sf">#sf</a> Slack channel<br />
-          <a href="http://www.gsa.gov/portal/category/107015">Building Services contact info</a>
+        <a href="https://gsa-tts.slack.com/messages/@eth">Ethan Heppner</a>
```

Comment:
@afeld @its-a-lisa tend to agree in the handbook we should have a hard rule that contact information can only be a google groups email or slack channel to avoid content maintenance.

Reply:
Thirded. @ethanheppner Thoughts about what to change this to?

Reply:
"Office management" has always been sort of a loosely defined responsibility, and I agree I
don't think it makes sense for those duties to fall to any one person in particular-- there really isn't much support I can provide personally to #sf in my current role.

I think the current content is more reflective of the current reality-- folks should discuss local office management issues in #sf first, with #teamops as a point of escalation for any issues that can't be resolved locally.

Reply:
@JJediny agree -- added this to the #handbook-updates

Are there any other 'hard rules' that we have for the handbook?

Reply:
So based on @ethanheppner comments above, should we close this without merging?